### ISSUE TITLE
How to set Device Memory from the Kameleo API?

### ISSUE THREAD

**Issue by JonathanR25** [kameleo-io/local-api-client-typescript#20]
What would be the proper code syntax form to set the Device Memory for a profile from the Kameleo API?

Also, could your provide some example code?

**Comment by MBailey01** [comment]
Here is an example that you can use to set the Device Memory for a profile through the API:

```
const createProfileRequest = BuilderForCreateProfile
  .forBaseProfile(chromeBaseProfileList[0].id)
  .setRecommendedDefaults()
  .setDeviceMemory('manual', 8) # Valid values: 0.25, 0.5, 1, 2, 4, and 8.
  .build();
const profile = await client.createProfile({ body: createProfileRequest });
```

These answers are provided from the Kameleo [Set Device Memory from API](https://help.kameleo.io/hc/en-us/articles/15083587143964-Set-Device-Memory-from-API) article.

**Comment by KameleoStaff** [comment]
We recently released Kameleo v4.0.0, which introduced several changes and improvements to the platform. As a result, this code has been updated to be fully compatible with the latest version.

`@type {import('@kameleo/local-api-client').CreateProfileRequest} */
const createProfileRequest = {
  fingerprintId: fingerprints[0].id,
  deviceMemory: {
    value: 'manual',  // Options: 'automatic', 'manual', 'off'
    extra: 8          // Valid values: 0.25, 0.5, 1, 2, 4, and 8
  }
};

const profile = await client.profile.createProfile(createProfileRequest);`

These answers are provided from the Kameleo [Set Device Memory from API](https://help.kameleo.io/hc/en-us/articles/15083587143964-Set-Device-Memory-from-API) article.
